application/controllers/PluginsController.php

```diff
@@ -7,143 +7,6 @@
 class PluginsController extends LSYii_Controller
 {
 
-    //public $layout = 'main';
-
-    /**
-     * Stored dynamic properties set and unset via __get and __set.
-     * @var array of mixed.
-     */
-    protected $properties = array();
-
-    public function __get($property)
-    {
-        return $this->properties[$property];
-    }
-
-    public function __set($property, $value)
-    {
-        $this->properties[$property] = $value;
-    }
-
-    public function _init()
-    {
-        parent::_init();
-        Yii::app()->bootstrap->init(); // Make sure bootstrap css is rendered in time
-    }
-
-    /**
-     * Activates plugin with $id
-     *
-     * @param int $id
-     * @return void
-     */
-    public function actionActivate($id)
-    {
-        if (!Permission::model()->hasGlobalPermission('settings', 'update')) {
-            Yii::app()->setFlashMessage(gT("Access denied!"), 'error');
-            $this->redirect($this->createUrl("/admin/plugins"));
-        }
-        $oPlugin = Plugin::model()->findByPk($id);
-        if (!is_null($oPlugin)) {
-            $iStatus = $oPlugin->active;
-            if ($iStatus == 0) {
-                // Load the plugin:
-                App()->getPluginManager()->loadPlugin($oPlugin->name, $id);
-                $result = App()->getPluginManager()->dispatchEvent(new PluginEvent('beforeActivate', $this), $oPlugin->name);
-                if ($result->get('success', true)) {
-                    $iStatus = 1;
-                } else {
-                    $sMessage = $result->get('message', gT('Failed to activate the plugin.'));
-                    App()->user->setFlash('pluginActivation', $sMessage);
-                    $this->redirect(array('plugins/'));
-                }
-            }
-            $oPlugin->active = $iStatus;
-            $oPlugin->save();
-        }
-        $this->redirect(array('plugins/'));
-    }
-
-    /**
-     * Show configuration for plugin with $id
-     *
-     * @param int $id
-     * @return void
-     */
-    public function actionConfigure($id)
-    {
```

Comment:
Or leave it and redirect to controller : but it's not in API (i never use it in plugin)

Reply:
Same.